wg-advisors/why/why_release_process.md

```diff
@@ -0,0 +1,30 @@
+# Release process
+
+## Summary 
+
+ASF project release software following a well defined process involving PMC members voting on release
+candidates, publishing the artifacts in the ASF distribution system, and announcing the release to the
+community.
+
+## Links to relevant documents
+
+* [Release Policy](https://www.apache.org/dev/release.html) 
+* [Release Distribution Policy](https://www.apache.org/dev/release-distribution.html)
+
+## Why are we doing it?
+
+We are doing it because this is what the ASF is doing. ASF delivers "Software for the public good"
+and the act of releasing software is a key part of that. The process is designed to ensure that the
+software is properly vetted and that the community is aware of the release and that the process of
+releasing software is transparent, open and secure. Releasing the software is a Legal Act of the 
+Foundation and it has consequences for the Foundation as software released by the Foundation is
+the main way the Foundation interacts with the public and can be hold accountable for the software
+it releases.
+
+## Is it mandatory and what are conditions?
+
+The process is mandatory for all ASF projects. The process is the same for all projects. This 
+
+## Are there variations for different projects?
+
+Not really. The process is the same for all projects.
```

Comment:
> some projects consider the RM as an implicit +1

Here is a patch to explain no RM implicit +1 https://github.com/apache/www-site/pull/338.

Reply:
Yes. Worth mentioning the variations.

But I think going into oo many details is not a good idea. It will be of coure always an art how "deep" we should describe it here, but the rule of thumb I think such document should not go into details where the linked document cover them, it should be a one-pager max and it should mostly focus on the most important "why's" - providing a very broad context that is otherwise distiributed among several documents, heads of people, discussions etc. etc. 

I think it serves two purposes:

* for the advisor - so that they can always come back here and immediately see - hey, why actually we are doing it ?  - a... here it is - and go into details if needed

* for the advised ones - to be assured that what we are advising them is not a "cargo cult" and tha we have very good reason that you can read about it here why we are doing it - without having to go through all the piles of documents and discussions or asking others. 

I will expand it a little bit soon.

Reply:
I added a short description describing those variations - with the goal to make it clearer for both "purposes". I hope it's better now.